### ISSUE TITLE
Perform a list of files from a folder (select) throws error and crashes

### ISSUE THREAD

**Issue by MM4096** [MM4096/MIDI-Controller#6]
Error:
```
Traceback (most recent call last):
  File "/home/coding/dev/repo/midi_project/curses_main.py", line 785, in <module>
    wrapper(menu)
  File "/usr/lib/python3.12/curses/__init__.py", line 94, in wrapper
    return func(stdscr, *args, **kwds)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/coding/dev/repo/midi_project/curses_main.py", line 774, in menu
    performance_mode(stdscr)
  File "/home/coding/dev/repo/midi_project/curses_main.py", line 665, in performance_mode
    this_patch_list = patcher.parse_patch_from_file(current_file_path)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/coding/dev/repo/midi_project/modules/patcher.py", line 80, in parse_patch_from_file
    with open(file_path, "r") as file:
         ^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '{patch_name}.midipatch'
```

**Comment by MM4096** [comment]
will be fixed in 0.4.2, **not** 0.4.1 (more important things to fix)

**Comment by MM4096** [comment]
Fixed in 0.4.2
